ynput/OpenPype
# Option to start versioning from 0

## Brief description
Enables v000 as first version

## Description
Modified all bits i could find on version to allow for 0 as first version. workfile gui also shows 0 as an option.

## Additional info
Ftrack had problems with version 0 in its API, solution was to cast 0 as string before committing the assetversion to db. Seem to work as intended.

## Testing notes:
1. Open Nuke
2. see that the first version of workfile can be set at 0 or is already built as 0 if you have workfile builder active
3. Publish and see v000 on ftrack.
Tested on padding 3 and 4

## Tested hosts
- [x] Nuke (render)
- [x] Maya (pointcache, review)
- [ ] Nukestudio
- [ ] 3dsmax
- [ ] AE
- [x] Tray Publisher

For now nuke, traypublisher and maya pass testing (limited to info in parenthesis)
